File: Jack.Presentation/Controllers/MVC/SacolasController.vb
Imports System.Web.Mvc

Namespace Controllers.MVC
    Public Class SacolasController
        Inherits Controller

        ' GET: Sacolas
        Function Index() As ActionResult
            Return View()
        End Function

        Function Livres() As ActionResult
            Return View()
        End Function

    End Class
End Namespace

File: Jack.Presentation.Web/Controllers/SacolasController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web.Mvc;
using Application = Jack.Application;
using Model = Jack.Model;

namespace Controllers.MVC
{
    public class SacolasController : Controller
	{

		// GET: Sacolas
		public ActionResult Index()
		{
			return View();
		}

		public ActionResult Livres()
		{
			return View();
		}

	}
}

